src/cache.js

```diff
@@ -0,0 +1,82 @@
+import run from './run';
+import { generateCloneTaskDefinition, generateRepoCacheTaskDefinition } from './task';
+
+// import cache config
+
+async function main(argv) {
+    var clones = [
+        'https://github.com/lightsofapollo/build-mozharness',
+        'https://hg.mozilla.org/build/mozharness',
+        'https://hg.mozilla.org/build/tools',
+        'https://hg.mozilla.org/mozilla-central',
+        'https://hg.mozilla.org/mozilla-central/',
+        'https://hg.mozilla.org/integration/b2g-inbound/',
+        'https://hg.mozilla.org/integration/b2g-inbound',
+        'https://hg.mozilla.org/integration/mozilla-inbound',
+        'https://hg.mozilla.org/integration/mozilla-inbound/',
+        'https://hg.mozilla.org/integration/gaia',
+        'https://hg.mozilla.org/integration/gaia-central',
+        'http://hg.mozilla.org/integration/fx-team/',
+        'http://hg.mozilla.org/integration/fx-team',
+        'https://git.mozilla.org/b2g/B2G.git',
+        'https://hg.mozilla.org/integration/gaia-central',
+        'https://hg.mozilla.org/integration/gaia-central/',
+        'https://git.mozilla.org/b2g/B2G',
+        'https://github.com/mozilla-b2g/gaia',
+        'https://github.com/mozilla-b2g/B2G',
+        'https://github.com/mozilla-b2g/B2G-manifest',
+        'https://github.com/mozilla-b2g/moztt',
+        'https://github.com/mozilla/gecko-dev'
+    ];
+
+
+    var emulators = [
+        ['emulator_url', 'emulator-ics'],
```

Comment:
since this is the same for all emulators, and we iterate through these separately from other things, do we need to include the emulator URL for each of them?